src/redux/types.js

```diff
@@ -0,0 +1,4 @@
+export const LOGIN_SUCCESS = 'LOGIN_SUCCESS';
+export const LOGIN_FAIL = 'LOGIN_FAIL';
+export const LOGOUT = 'LOGOUT';
+export const SET_LOADING = 'SET_LOADING';
```

Comment:
Make it more generic, something to denote login initiation, as there would be additional settings added other then setting loading.